modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IdentityUserOrganizationUnit.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using Volo.Abp.Domain.Entities.Auditing;
+using Volo.Abp.MultiTenancy;
+
+namespace Volo.Abp.Identity
+{
+    /// <summary>
+    /// Represents membership of a User to an OU.
+    /// </summary>
+    public class IdentityUserOrganizationUnit : CreationAuditedEntity<Guid>, IMultiTenant, ISoftDelete
```

Comment:
Can `IdentityUserOrganizationUnit` and `OrganizationUnitRole ` inherit from `Entity`?`

They do not need a primary key. Such as `IdentityUserRole`

Reply:
You are right. I fixed. Thank you.